version.txt

```diff
@@ -1 +1 @@
-0.4
+1.0
```

Comment:
+1 on using non-zero major version numbers for packages in production.

It looks like your CHANGELOG.md is still boilerplate -- time to start populating it or towncrier or something? The former is the standard; Tom's pushing for the latter.